### PR TITLE
[dash] Moving more files around and fixing broken links that result

### PR DESCRIPTION
**Pull request by Sfshaza**
@chalin

### CODE REVIEW

File: src/_data/catalog/index.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
   },
   {
     "name": "Animation and Motion",
-    "description": "Bring animations to your app. Check out <a href=\"/animations/\">Animations</a> in Flutter for an overview.",
+    "description": "Bring animations to your app. Check out <a href=\"/development/ui/animations\">Animations</a> in Flutter for an overview.",
```
Comment (chalin) [inline src/_data/catalog/index.json]: Hmm, is this meant to be the page description? I don't think that we should have links in our descriptions.

**Comment by chalin** [comment]
Hey Shams: Travis is still reporting some broken links.

File: src/_data/catalog/index.json (hunk 1 above)
Comment (Sfshaza) [reply src/_data/catalog/index.json]: No, this was the source of great annoyance. It took me forever to find this broken link. I don't know who put this there but I'd be fine with removing it.

**Comment by Sfshaza** [comment]
Patrice and I have both signed the CLA.

**Pull request merged**